r/tests/testthat/test-parquet.R

```diff
@@ -49,3 +49,62 @@ test_that("read_parquet() with raw data", {
   df <- read_parquet(test_raw)
   expect_identical(dim(df), c(10L, 11L))
 })
+
+test_that("write_parquet() handles various compression= specs", {
+  tab <- Table$create(x1 = 1:5, x2 = 1:5, y = 1:5)
+  tf <- tempfile()
+
+  # a single string
+  expect_error(write_parquet(tab, tf, compression = "snappy"), NA)
```

Comment:
Isn't `expect_error(..., NA)` unnecessary? The tests will exit on error, so the test suite will fail.

Also, IMO at least one of these should read the file back in to confirm that we wrote valid data. I'm guessing we can't assert details of the Parquet file's encoding and are trusting the C++ test coverage for that, but we should at least confirm that we've written out a valid Parquet file with the expected contents.